solutions/account-infrastructure-base/variables.tf

```diff
@@ -345,3 +345,75 @@ variable "trusted_profile_roles" {
   description = "A list of roles given to the trusted profile."
   default     = ["Administrator"]
 }
+
+variable "provision_cbr" {
+  type        = bool
+  description = "Whether to enable the creation of context-based restriction rules and zones in the solution."
+  default     = false
+}
+
+variable "cbr_allow_cos_to_kms" {
+  type        = bool
+  description = "Whether to enable the rule that allows Object Storage to access the key management service. Default is true."
+  default     = true
```

Comment:
As per my understanding, it will be in effect if `provision_cbr` is true. If it is so, then we should clearly say that in description of variable like...`Default is true if input variable 'provision_cbr' is set to true`.

Same for other variables too.

Reply:
Updated all variables to clarify